docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -21,7 +21,6 @@ The following table lists the available UI components:
 | [Popup](./nui-components/Popup.md) | A popup is used as a popup window. | `Popup`                |
```

Comment:
Just for checking. In this page we have below sentence.

UI components are interactive components for layout and scrolling the user interface. NUI provides UI components, such as buttons, scroll view, table view, text controls, image view, flex container, slider, and video view.

Don't we need to update this sentence?

Reply:
> Just for checking. In this page we have below sentence.
> 
> UI components are interactive components for layout and scrolling the user interface. NUI provides UI components, such as buttons, scroll view, table view, text controls, image view, flex container, slider, and video view.
> 
> Don't we need to update this sentence?

OK I removed "scroll view, "

Reply:
Thanks for the update. However I also doubt the ".. and scrolling the user interface".

Reply:
> Thanks for the update. However I also doubt the ".. and scrolling the user interface".

I think it's better after removing "scrolling the". How do you think?

Reply:
I agree with you. Thanks for the update.